CONTRIBUTING.md

```diff
@@ -53,18 +71,33 @@ $ cargo test --test <test_name> --features debug
 2. Push your changes back to your fork (`git push origin $your-branch`)
 3. Create a pull request against `master`! (You can also create the pull request first, and we'll merge when ready. This a good way to discuss proposed changes.)
 
-### Goals
+## Conditions for fulfilling a bounty:
 
-There are a few goals of `clap` that I'd like to maintain throughout contributions. If your proposed changes break, or go against any of these goals we'll discuss the changes further before merging (but will *not* be ignored, all contributes are welcome!). These are by no means hard-and-fast rules, as I'm no expert and break them myself from time to time (even if by mistake or ignorance :P).
+1. You should make a pull request which fixes the issue the bounty was promised for
+2. The pull request should be merged by one of the maintainers
 
-* Remain backwards compatible when possible
-  - If backwards compatibility *must* be broken, use deprecation warnings if at all possible before removing legacy code
-  - This does not apply for security concerns
-* Parse arguments quickly
-  - Parsing of arguments shouldn't slow down usage of the main program
-  - This is also true of generating help and usage information (although *slightly* less stringent, as the program is about to exit)
-* Try to be cognizant of memory usage
-  - Once parsing is complete, the memory footprint of `clap` should be low since the main program is the star of the show
-* `panic!` on *developer* error
-  (e.g. [apps](https://github.com/clap-rs/clap/blob/62eff1f8d3394cef819b4aa7b23a1032fc584f03/src/build/app/debug_asserts.rs) and [args](https://github.com/clap-rs/clap/blob/62eff1f8d3394cef819b4aa7b23a1032fc584f03/src/build/arg/debug_asserts.rs)),
-  exit gracefully on *end-user* error
+### Below are the steps to redeem a bounty:
+
+1. Go to https://opencollective.com/clap/expenses/new.
+2. Select **Invoice**.
+3. Enter **Expense Title** as "Issue Bounty".
+4. In **Description**, link the issue you are redeeming _(Ex: `https://github.com/clap-rs/clap/issues/1464`)_
+5. In **Amount**, write the amount that the issue promised _(Ex: 10)_
+6. Fill payment information and submit
+7. Wait for us to approve it
+
+### Can I forgo the bounty?
+
+Yes, you can. In that case, you don't have to do anything except writing a
+comment on the issue saying that I do. The bounty will be reassigned to another
+issue.
+
+## Specific Tasks
+
+### Updating MSRV
+
+Search for `MSRV`, for example
+```bash
+$ rg --hidden MSRV
```

Comment:
```suggestion
$ rg --hidden 1.54.0
```

I am assuming you meant an actual version as an example?

Reply:
No, each MSRV is annotated with an MSRV comment.